packages/voice/examples/recorder/tsconfig.json

```diff
@@ -3,6 +3,7 @@
 	"compilerOptions": {
 		"baseUrl": ".",
 		"outDir": "dist",
+		"skipLibCheck": true,
```

Comment:
This is due to the typescript error in prism-media:
```
node_modules/prism-media/dist/ffmpeg/index.d.ts:3:32 - error TS7016: Could not find a declaration file for module 'duplex-child-process'. '/Users/tony/Projects/discord.js/packages/voice/examples/recorder/node_modules/duplex-child-process/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/duplex-child-process` if it exists or add a new declaration (.d.ts) file containing `declare module 'duplex-child-process';`

3 import DuplexChildProcess from 'duplex-child-process';

```

Reply:
Should I file an issue in prism-media repo?

Reply:
Sure